ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -16,10 +16,10 @@ REGION="${LEASED_RESOURCE}"
 
 # Generate working availability zones from the region
 mapfile -t AVAILABILITY_ZONES < <(aws --region "${REGION}" ec2 describe-availability-zones | jq -r '.AvailabilityZones[] | select(.State == "available") | .ZoneName' | sort -u)
-# Generate availability zones with the biggest instance type required
-mapfile -t XLARGE_ZONES < <(aws --region "${REGION}" ec2 describe-instance-type-offerings --location-type availability-zone --filters Name=instance-type,Values=m5.8xlarge | jq -r '.InstanceTypeOfferings[].Location' | sort -u)
+# Generate availability zones with all instance types required
+mapfile -t INSTANCE_ZONES < <(aws --region "${REGION}" ec2 describe-instance-type-offerings --location-type availability-zone --filters Name=instance-type,Values=m5.8xlarge,m5.4xlarge,m5.2xlarge,m5.xlarge,m4.xlarge | jq -r '.InstanceTypeOfferings[].Location' | sort -u)
```

Comment:
Would it be easy to get the actual instance types that are going to be used?

Reply:
That's a good point. I don't see why not.